app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -107,38 +152,41 @@ class RefereeProfile extends Component {
       )
     }
 
+    if (edit) {
+      return <RefereeProfileForm defaultValues={referee} onSubmit={this.save} />
```

Comment:
I love this form, I think for better usability let's make this component the content of a Modal, so the user can see their data change in real time.

Reply:
Great idea!